client/a8c-for-agencies/data/team/use-resend-member-invite.ts

```diff
@@ -0,0 +1,44 @@
+import { useMutation, UseMutationOptions, UseMutationResult } from '@tanstack/react-query';
+import wpcom from 'calypso/lib/wp';
+import { useSelector } from 'calypso/state';
+import { getActiveAgencyId } from 'calypso/state/a8c-for-agencies/agency/selectors';
+
+interface APIError {
+	status: number;
+	code: string | null;
+	message: string;
+}
+
+export interface resendMemberInviteParams {
+	id: number;
+}
+
+interface APIResponse {
+	success: boolean;
+}
+
+function resendMemberInviteMutation(
+	params: resendMemberInviteParams,
+	agencyId?: number
+): Promise< APIResponse > {
+	if ( ! agencyId ) {
+		throw new Error( 'Agency ID is required to assign a license' );
```

Comment:
This should say `Agency ID is required to resend invite`

Reply:
Good catch! I forgot to remove this after copy/pasting from a different hook.